.travis/before_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
The generally recommended practice is to do
```` sh
#!/usr/bin/env bash
````
This looks for 'bash' in the path, rather than assuming it's /bin/bash (which only holds for some Linuces, and usually not for all non-Linuces).

An alternative can be
```` sh
#!/bin/sh
````
Which is the default shell everywhere (although not sure it's actually required by some standard to exist).
While Linuces typically use 'bash' as 'sh' replacement, other systems (eg BSD) use the real sh shell. Your script thus may not contain bash-specific code in this case.

Since this code is specific for Travis, I see no real harm in doing it like you propose.